### DISCUSSION TITLE
CNCF TAG-Runtime Discussion/Presentation (?)

### DISCUSSION THREAD

**Discussion by raravena80** [Netcracker/KubeMarine#373] [General]
Hello KubeMarine team,

I'm one of the co-chairs of the CNCF TAG-Runtime. I believe you had an agenda item to present the project in our meeting. I'm following up, I think it would be great for you to present/discuss the project in the future.

Feel free to add it to our [agenda](https://bit.ly/cncf-tag-runtime-meeting-notes) or reach out to me (raravena80 at gmail.com) at your convenience. 

Thanks!

**Comment by koryaga** [comment]
Hello @raravena80, 
Thank you for your interest provided to our product. We will get back to you next week with some input for discussion.
Regards
@proivk

**Comment by koryaga** [comment]
Hi @raravena80, we are ready to communicate and meet. Based on the document you provided, it appears that the next available slot is April 6th. Would it be possible to arrange a meeting at that time?

**Comment by raravena80** [comment]
@koryaga there are some other agenda items on April 4th.  Would it be possible to present on May 4th, 2023?  If yes, feel free to add it to the [agenda](https://bit.ly/cncf-tag-runtime-meeting-notes) Thanks!

**Reply by koryaga** [reply]
Applied to May 4th, 2023. Thank you.

**Comment by koryaga** [comment]
Rejected https://github.com/cncf/sandbox/issues/16
